src/main/java/com/dabsquared/gitlabjenkins/trigger/handler/pipeline/PipelineHookTriggerHandlerImpl.java

```diff
@@ -132,7 +132,9 @@ protected CauseData retrieveCauseData(PipelineHook hook) {
                                 ? ""
                                 : hook.getRepository().getGitSshUrl())
                 .withSourceRepoHttpUrl(
-                        hook.getRepository() == null ? "" : hook.getRepository().getGitHttpUrl())
+                        hook.getRepository() == null || hook.getRepository() == null
```

Comment:
Why is `hook.getRepository() == null` repeated here?

Reply:
This commit is made by some other contributor. When I opened PR this commit came automatically still I'll make the changes in this code.

Reply:
That is fixed on the master branch.  You need to merge the master branch into this branch.

Reply:
Looks like it was from you merged 3 hours ago @AniketNS : https://github.com/jenkinsci/gitlab-plugin/commit/6ac7bd7a585d55d4b64a04be3e351ba66a5cc945

Reply:
Yeah, I mistakenly reverted my code to the old commit before this code was committed. I'm very sorry about that. I'm looking into this. I'll correct it or have to open the new PR for this. Would that be okay?